VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitDiscussionFragment.java

```diff
@@ -0,0 +1,77 @@
+package org.edx.mobile.view;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentTransaction;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import org.edx.mobile.R;
+import org.edx.mobile.model.api.EnrolledCoursesResponse;
+import org.edx.mobile.model.course.CourseComponent;
+import org.edx.mobile.model.course.DiscussionBlockModel;
+import org.edx.mobile.services.ViewPagerDownloadManager;
+
+public class CourseUnitDiscussionFragment extends CourseUnitFragment {
```

Comment:
It seems like the only reason that `CourseDiscussionPostsThreadFragment` could not be used directly is because all the course unit fragments are expected to extend `CourseUnitFragment`. However, that base class doesn't seem to provide any critical function, and could potentially be removed entirely.